### PR TITLE
Add support for "Object" type in schema

### PR DESCRIPTION
**Pull request by mmandlis**

### CODE REVIEW

File: converters/jsonldToManifest.js [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-var supportedTypes = ["Text", "URL", "Number", "Boolean"];
+var supportedTypes = ["Text", "URL", "Number", "Boolean", "Object"];
```
Comment (shans) [inline converters/jsonldToManifest.js]: don't need this here as Object isn't a schema.org type.

File: runtime/schema.js [hunk 2]
```diff
@@ -76,6 +76,8 @@ class Schema {
           return [fieldType, 'number'];
         case 'Boolean':
           return [fieldType, 'boolean']
+        case 'Object':
+          return [fieldType, 'Object']
```
Comment (shans) [inline runtime/schema.js]: lowercase 'object'

File: converters/jsonldToManifest.js (hunk 1 above)
Comment (mmandlis) [reply converters/jsonldToManifest.js]: done

File: runtime/schema.js (hunk 2 above)
Comment (mmandlis) [reply runtime/schema.js]: done

**Pull request merged**